src/main/java/pl/psnc/dei/service/ImportPackageService.java

```diff
@@ -149,4 +149,21 @@ public ImportReport getStatusWithFailure(String importName) throws NotFoundExcep
         }
         return ImportReport.from(anImport.getStatus(), anImport.getFailures());
     }
+
+    public Import addRecordsToImport(String importName, List<Record> records) throws NotFoundException {
+        log.info("Adding records to import {}, records {}", importName, records);
+        Import anImport = importsRepository.findImportByName(importName);
+        if (anImport == null) {
+            throw new NotFoundException("Import not found");
+        }
+        records.forEach(record -> {
```

Comment:
This part of code is duplicated in create and update. It's a good candidate for extracting a method.